crates/oxc_allocator/src/convert.rs

```diff
@@ -0,0 +1,60 @@
+#![allow(clippy::inline_always)]
+
+use crate::{Allocator, Box};
+
+pub trait FromIn<'a, T> {
```

Comment:
Consider adding doc comments

Reply:
I'll do it before marking the PR for review, I usually start by making it work then go back and do the cleanup/documentation.